source/nodejs/adaptivecards/src/strings.ts

```diff
@@ -0,0 +1,37 @@
+export class Strings {
+    static readonly errors = {
+        unknownElementType: "Unknown element type \"{0}\". Fallback will be used if present.",
```

Comment:
I think it would improve legibility to use `'` instead of `"` when the string has embedded `"`s :)

```suggestion
        unknownElementType: 'Unknown element type "{0}". Fallback will be used if present.',
```

Reply:
I tend to never use `'`. But anyway, I am making these resource strings functions, and this is addressed in the process.